contracts/src/registry/EnhancedAccessControl.sol

```diff
@@ -188,23 +188,24 @@ abstract contract EnhancedAccessControl is Context, ERC165 {
      * @param srcAccount The account to copy roles from.
      * @param dstAccount The account to copy roles to.
      */
-    function _copyRoles(bytes32 resource, address srcAccount, address dstAccount) internal virtual {
+    function _copyRoles(bytes32 resource, address srcAccount, address dstAccount, bool enableCallbacks) internal virtual {
         uint256 srcRoles = _roles[resource][srcAccount];
-        _grantRoles(resource, srcRoles, dstAccount);
+        _grantRoles(resource, srcRoles, dstAccount, enableCallbacks);
     }
 
     /**
      * @dev Grants multiple roles to `account`.
      */
-    function _grantRoles(bytes32 resource, uint256 roleBitmap, address account) internal virtual returns (bool) {
+    function _grantRoles(bytes32 resource, uint256 roleBitmap, address account, bool enableCallbacks) internal virtual returns (bool) {
```

Comment:
Perhaps `executeCallbacks` instead of `enableCallbacks`; 'enable' implies a persistent change.